coffee-chats/src/main/webapp/src/requests/tags.ts

```diff
@@ -0,0 +1,13 @@
+const tagsURL = '/api/tags';
+
+export const fetchTags = () => {
+  return fetch(tagsURL)
+  .then((response) => response.json())
```

Comment:
Would await work here?

Reply:
Yes, is it preferred over promise chaining?

Reply:
It's just an easier syntax to do the same, because it's more modern. But style guide doesn't say either way is better, so w/e